projects/js-packages/publicize-components/src/components/generated-image-preview/index.js

```diff
@@ -23,24 +24,29 @@ export default function GeneratedImagePreview() {
 	const [ isLoading, setIsLoading ] = useState( true );
 
 	const { customText, imageType, imageId, template } = useImageGeneratorConfig();
-	const { getMedia } = select( 'core' );
-	const { title, featuredImage } = useSelect( _select => ( {
-		title: _select( editorStore ).getEditedPostAttribute( 'title' ),
-		featuredImage: _select( editorStore ).getEditedPostAttribute( 'featured_media' ),
+	const { title, featuredImage } = useSelect( select => ( {
+		title: select( editorStore ).getEditedPostAttribute( 'title' ),
+		featuredImage: select( editorStore ).getEditedPostAttribute( 'featured_media' ),
 	} ) );
 
-	const imageTitle = useMemo( () => customText || title || ' ', [ customText, title ] );
-	const imageTitleRef = useRef( imageTitle );
-	const getImageUrl = useCallback( () => {
-		if ( imageType === 'featured' ) {
-			return getMediaSourceUrl( getMedia( featuredImage ) );
+	const imageUrl = useSelect( select => {
+		const getMedia = select( 'core' ).getMedia;
+		if ( imageType === 'none' ) {
+			return null;
 		}
 		if ( imageType === 'custom' ) {
 			return getMediaSourceUrl( getMedia( imageId ) );
```

Comment:
Is there any chance that this could still be loading too? Maybe if it was previously saved with a custom image and then we've got a hard refresh/cold start of the editor?